packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -144,7 +144,7 @@ const COMMON_TESTS = new Map([
 ]);
 
 // prettier-ignore
-const TESTS = new Map([
+const TESTS = [
```

Comment:
In a previous patch for the expiring CC reminder emails, TESTS was treated as an array in a `for...in`; this fixes that issue by changing TESTS to an array.  This allows us to have more than one test per mailer method.